mappings/net/minecraft/world/storage/WorldSaveStorage.mapping

```diff
@@ -82,9 +101,15 @@ CLASS net/minecraft/unmapped/C_bpwagnit net/minecraft/world/storage/WorldSaveSto
 			ARG 1 key
 		CLASS C_broyznts
 			METHOD postVisitDirectory postVisitDirectory (Ljava/lang/Object;Ljava/io/IOException;)Ljava/nio/file/FileVisitResult;
+				ARG 1 path
+				ARG 2 iOException
```

Comment:
```suggestion
				ARG 2 ioException
```
Another thing I'm not quite sure on, but based on [abbreviation conventions](https://github.com/QuiltMC/quilt-mappings/blob/1.20-pre6/CONVENTIONS.md#abbreviations)
> Treat acronyms as single words rather than capitalizing every letter. This improves readability (compare JsonObject and JSONObject) and it's consistent with Mojang naming (a known name is NbtIo).

However since this is a name for Java's `IOException`, that may take precedence, in which case your original name is correct. 

Again, other reviewers please comment.

Reply:
imo `ioException` works better.